toqito/matrix_ops/tensor_comb.py

```diff
@@ -30,42 +35,64 @@ def tensor_comb(states: list[np.ndarray], k: int) -> dict:
 
     .. jupyter-execute::
 
-        import numpy as np
-        from toqito.matrix_ops import tensor_comb
+     from toqito.matrix_ops import tensor_comb
+     import numpy as np
 
-        e_0 = np.array([1, 0])
-        e_1 = np.array([0, 1])
-        result = tensor_comb([e_0, e_1], 2)
+     e_0 = np.array([1, 0])
+     e_1 = np.array([0, 1])
 
-        for key, mat in result.items():
-            print(f"tensor_comb{key} =\n{mat}\n")
+     result = tensor_comb([e_0, e_1], 2, mode="injective", density_matrix="density_matrix")
+
+     for key, mat in result.items():
+         print(f"tensor_comb{key} =\n{mat}\n")
 
     References
     ==========
     .. bibliography::
         :filter: docname in docnames
 
     :raises ValueError: If the input list of states is empty.
-    :param states: A list of quantum state vectors represented as numpy arrays.
-    :param k: The length of the sequence for generating tensor products.
-    :return: A dictionary where:
-
-        - Keys represent sequences (as tuples) of quantum state indices,
-        - Values are density matrices corresponding to the tensor product of
-          the state vectors for the sequence.
+    :param states: A list of state vectors.
+    :param k: The length of the sequence.
+    :param mode: Determines the type of sequences.-`non-injective`: Allows repetitions in
+           sequences.-`injective`: Ensures sequences are injective (no repetitions).-`diagonal`: Only
+           sequences with repeated indices (diagonal elements).
```

Comment:
Previous comment about this is still unresolved. You cannot add a list here.

If you are using `-`, this is an attempt at creating a list.